src/ResourceManager/Insights/Commands.Insights/LogsCmdletBase.cs

```diff
@@ -36,11 +36,11 @@ public abstract class LogsCmdletBase : MonitorClientCmdletBase
         private const int MaxNumberOfReturnedRecords = 1000;
         private int MaxRecords = 0;
 
-        internal const string SubscriptionLevelName = "GetBySubscription";
```

Comment:
@gucalder please revert the change made to these parameter set names - we want parameter sets to be singular strings with pascal casing. I'm not sure if this was just changed because of a merge or not.

Reply:
Sorry I did not see this before I pushed the changes for the other comments.
Do you need me to make the name of the constant back to what it was?  (I will need to change at least one of them since ResourceGroupName is the name of a parameter now.

Reply:
@gucalder sorry, I made this comment a few hours after when I was looking at a couple of last minute things on this PR. We've received feedback that parameter set names should be a string with no spaces instead of the space-delimited string you are currently using. I recently merged a PR where we fix this in all cmdlets who were using them. If it wouldn't be too much trouble, would you mind revert the set names back to what they were originally, but updating them to reflect the changes you made? That would be great ☺️

Reply:
Ok. 
The content of each of those constants is now a single string, Pascal casing. I also updated the documentation to reflect that.

Reply:
@gucalder great, let me know when you push that update out

Reply:
I checked the files and I have already pushed all those changes for parameter groups names: they are all single strings, Pascal casing.